dbsnp-import/src/main/java/uk/ac/ebi/eva/server/models/ProgressReport.java

```diff
@@ -70,6 +70,14 @@
 
     private Date rsSynonymsImportedDate;
 
+    private Long importedIds;
+
+    private Long totalIdsDbsnp;
+
+    private Long importedSynIds;
+
+    private Long totalSynIdsDbsnp;
```

Comment:
IMO, we shouldn't use abbreviations like Syn

Reply:
Maybe I would drop the last Dbsnp as well, although I can concede it helps understanding that those are the ones in dbsnp, and the imported are in EVA